examples/lib/defaultHtmlSequencerUi.js

```diff
@@ -31,23 +32,19 @@ function DefaultHtmlSequencerUi(_sequencer, options) {
     var index = $(removeStepSel).index(this) + 1;
     sequencer.removeSteps(index).run();
     // remove from URL hash too
-    var urlHash = getUrlHashParameter("steps").split(",");
-    urlHash.splice(index - 1, 1);
-    setUrlHashParameter("steps", urlHash.join(","));
+    setUrlHashParameter("steps", sequencer.toString());
   }
 
   function addStepUi() {
     if ($(addStepSel + " select").val() == "none") return;
 
-    // add to URL hash too
-    var hash = getUrlHashParameter("steps") || "";
-    if (hash != "") hash += ",";
-    setUrlHashParameter("steps", hash + $(addStepSel + " select").val());
-
     var newStepName = $(addStepSel + " select").val();
     _sequencer
-      .addSteps(newStepName, options)
-      .run(null);
+    .addSteps(newStepName, options)
+    .run(null);
+
+    // add to URL hash too
+    setUrlHashParameter("steps", _sequencer.toString());
```

Comment:
And we're sure this'll overwrite the previous one? Great.

Reply:
Yeah!!